nncf/torch/sparsity/movement/functions.py

```diff
@@ -0,0 +1,29 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import torch
+
+from nncf.torch.dynamic_graph.patch_pytorch import register_operator
+from nncf.torch.functions import STThreshold
+
+@register_operator()
+def binary_mask_by_threshold(importance, threshold=0.5, sigmoid=False, max_percentile=0.98):
```

Comment:
develop unit test

Reply:
added in https://github.com/vuiseng9/nncf/pull/10/commits/704fe4e53768e7ba3fa5da474609e57c5d22f3dc